### DISCUSSION TITLE
RfC - Developer Training and Certification Program for django CMS

### DISCUSSION THREAD

**Discussion by macolo** [django-cms/django-cms#7882] [General]
## Why
- As django CMS Association I would like to grow the community and enable adoption of the product by providing a smooth onboarding experience for developers.

## What
- training course, coaching, certification, ...

## How
django CMS Association...
   - to nominate, endorse and recommend trusted service providers or individuals to act as coaches and instructors
   - to provide a certificate template
  
 nominated instructors can then offer (paid) courses and coaching

**Comment by ralfbraendli** [comment]
What target audience would make sense to start with?

- Django developer without Django CMS experience (Introduction to Django CMS for Django Developers)
- Web-Developer without any Django experience (Introduction to Django and Django CMS)
- Django developer with initial experience with Django CMS (In depth training for CMS Plugins and CMS Apps)
- Some other target audiences?

If we define a target audience, I can create an outline of topics for a first training.

I think an introduction to Django and Django CMS would be helpful for a company who are looking for Django / Django CMS developers but couldn't find anyone to hire a Web-Developer and bring it to a point he can work on Django CMS projects.

**Reply by macolo** [reply]
Hi @ralfbraendli "Django developer without Django CMS experience (Introduction to Django CMS for Django Developers)" would make most sense from the perspective of the django CMS community. The program would serve two purposes:
- certification (as a way for the certified to demonstrate expertise)
- training (to become proficient in django CMS)
